dbt/include/teradata/macros/materializations/incremental/incremental.sql

```diff
@@ -43,6 +43,11 @@
 
 {{ run_hooks(post_hooks, inside_transaction=True) }}
 
+-- implemented this for the grants feature
```

Comment:
hooks are running then grants are running later. Is this same as dbt core behaviour?

Reply:
We checked the behavior of dbt-core for hooks and grants and we have made the necessary changes to get aligned with dbt-core.

Reply:
@VarunSharma15 is this checked to seeds, snapshot aswell?

Reply:
Yes Satish, we have changes for other materializations as well.